cmd/node/main.go

```diff
@@ -428,6 +432,8 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		nodesConfig.StartTime = (ntpTime.Unix()/60 + 1) * 60
 	}
 
+	keyGenForBalances := signing.NewKeyGenerator(getSuiteForBalances())
```

Comment:
keyGenForBalances does not say what it is. you do not generate keys for balances but for accounts.

Reply:
renamed to keyGenForAccounts